src/components/AddObsBottomSheet/AddObsBottomSheet.tsx

```diff
@@ -0,0 +1,169 @@
+  navAndCloseBottomSheet: ( screen: string, params?: {
+    camera?: string
+  } ) => void;
+  hidden: boolean;
+}
+
+type ObsCreateItem = {
+    text?: string,
+    icon: string,
+    onPress: ( ) => void,
+    testID: string,
+    accessibilityLabel: string,
+    accessibilityHint: string
+}
+
+const majorVersionIOS = parseInt( String( Platform.Version ), 10 );
+const AI_CAMERA_SUPPORTED = ( Platform.OS === "ios" && majorVersionIOS >= 11 )
+  || ( Platform.OS === "android" && Platform.Version > 21 );
+
+const GREEN_CIRCLE_CLASS = "bg-inatGreen rounded-full h-[36px] w-[36px] mb-2";
+const ROW_CLASS = "flex-row justify-center space-x-4 w-full flex-1";
+
+const AddObsBottomSheet = ( {
+  closeBottomSheet, navAndCloseBottomSheet, hidden
+}: Props ) => {
+  const { t } = useTranslation( );
+
+  const prepareObsEdit = useStore( state => state.prepareObsEdit );
+
+  const obsCreateItems = useMemo( ( ) => ( {
+    aiCamera: {
+      text: t( "ID-with-AI-Camera" ),
+      icon: "aicamera",
+      onPress: ( ) => navAndCloseBottomSheet( "Camera", { camera: "AI" } ),
+      testID: "aicamera-button",
+      accessibilityLabel: t( "AI-Camera" ),
+      accessibilityHint: t( "Navigates-to-AI-camera" )
+    },
+    standardCamera: {
+      text: t( "Take-photos" ),
+      icon: "camera",
+      onPress: ( ) => navAndCloseBottomSheet( "Camera", { camera: "Standard" } ),
+      testID: "camera-button",
+      accessibilityLabel: t( "Camera" ),
+      accessibilityHint: t( "Navigates-to-camera" )
+    },
+    photoLibrary: {
+      text: t( "Upload-photos" ),
+      icon: "photo-library",
+      onPress: ( ) => navAndCloseBottomSheet( "PhotoLibrary" ),
+      testID: "import-media-button",
+      accessibilityLabel: t( "Photo-importer" ),
+      accessibilityHint: t( "Navigates-to-photo-importer" )
+    },
+    soundRecorder: {
+      text: t( "Record-a-sound" ),
+      icon: "microphone",
+      onPress: ( ) => navAndCloseBottomSheet( "SoundRecorder" ),
+      testID: "record-sound-button",
+      accessibilityLabel: t( "Sound-recorder" ),
+      accessibilityHint: t( "Navigates-to-sound-recorder" )
+    },
+    noEvidence: {
+      text: t( "Create-observation-with-no-evidence" ),
+      icon: "noevidence",
+      onPress: async ( ) => {
+        const newObservation = await Observation.new( );
+        prepareObsEdit( newObservation );
+        navAndCloseBottomSheet( "ObsEdit" );
+      },
+      testID: "observe-without-evidence-button",
+      accessibilityLabel: t( "Observation-with-no-evidence" ),
+      accessibilityHint: t( "Navigates-to-observation-edit-screen" )
+    }
+  } ), [
+    navAndCloseBottomSheet,
+    prepareObsEdit,
+    t
+  ] );
+
+  const renderAddObsIcon = ( {
+    accessibilityHint,
+    accessibilityLabel,
+    icon,
+    onPress,
+    testID,
+    text
+  }: ObsCreateItem ) => (
+    <Pressable
+      className="bg-white w-1/2 flex-column items-center py-4 rounded-lg flex-1 shadow-sm
+      shadow-black/25 active:opacity-50"
```

Comment:
Please make sure that these still work when we import from `styledComponent`: shadow-... active:...